### DISCUSSION TITLE
Frigate hardware acceleration

### DISCUSSION THREAD

**Discussion by helio58** [tteck/Proxmox#3540] [Help Needed]
Hi, thanks for this great script.

Im running frigate with hardware acceleration, it works fine same times for 3-12 hours,
Suddenly i cant reach to the frigate lxc or the proxmox  computer. 
I m running proxmox headless. Dont know how to see the errors.

Removing hardware acceleration works över a week.
Please any clue where skall i search?
Thanks

**Comment by chrislawso** [comment]
Hi, most logs are found in directory /var/log/ or you can review the error logs in the webgui at http://ip.add.re.ss:5000/logs

My system shows repeating error logs 'Ffmpeg process crashed unexpectedly' and I am uncertain if it is related to using hwaccel or not.

I am also experiencing some issues that sometimes take several hours to manifest with certain webpages ie birdseye and some cameras becoming unresponsive in live view and requiring reloading/refreshing webpage. Many times after refreshing the webpage the cameras will not display and I need to restart the lxc. 

I have tried without hwaccel and with hwaccel_args: preset-nvidia-h264

What hwaccel are you using?

Please continue to update this page with any more information you find.

Thank you

**Reply by helio58** [reply]
Thanks for responding, I think I have a different problem. I cant access the proxmox server nor the frigate webpage. cannot ping or ssh. The only solution is unplug the proxmox computer and start up again.
the hwaccel I have used is:
ffmpeg:
  hwaccel_args: preset-vaapi
For now I have removed the 2 lines in the proxmox container
lxc.mount.entry: /dev/dri dev/dri none bind,optional,create=dir
lxc.mount.entry: /dev/dri/renderD128 dev/dri/renderD128 none bind,optional,create=file
and it works just without hwaccel.
Thanks
